### ISSUE TITLE
Expand Preferences

### ISSUE THREAD

**Issue by Lunnaris01** [hhu-stups/bmoth#26]
Could include multiple styles for the editor and highlighting like IntelliJ provides them.
Feel free to leave any requests.
Could include multiple Preferences, for multiple Users/Projects.

blocked by #27

**Comment by wysiib** [comment]
One preference we definitely need is the minimum and maximum size of integers, used to distinguish NAT from NATURAL and INT from INTEGER.

**Comment by wysiib** [comment]
i think this issue is too general. we should split it into multiple ones for different preferences. otherwise, it will probably never be finished. Closing it as invalid.
